postgresSeed.js

```diff
@@ -1,31 +1,49 @@
 const fs = require('fs');
 const faker = require('faker');
 
+// 26595195
+
 const entryNum = 10000000;
+const reviewCount = 26595195;
+
+let reviewId = 0;
```

Comment:
Names are a little confusing. Maybe change the names or specify their meaning when you declare them?